pkgs/by-name/st/statping-ng/package.nix

```
@@ -0,0 +1,138 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  stdenv,
+  yarn,
+  nodejs,
+  go-rice,
+  go,
+  git,
+  cacert,
+  webpack-cli,
+  jq,
+}:
+buildGoModule rec {
+  pname = "statping-ng";
+  version = "0.91.0";
+
+  subPackages = [
+    "cmd/"
+  ];
+  src = fetchFromGitHub {
+    owner = "statping-ng";
+    repo = "statping-ng";
+    tag = "v${version}";
+    hash = "sha256-1jfllhy6/1SED9SEqtVQFF0i/K2POwM+kRhI9ZMFjvo=";
+  };
+  env = {
+    CYPRESS_INSTALL_BINARY = 0;
+  };
+  offlineCache = stdenv.mkDerivation {
+    name = "${pname}-${version}-yarn-offline-cache";
+    inherit src env;
+    nativeBuildInputs = [
+      yarn
+      nodejs
+      cacert
+      jq
+      go
+      git
+    ];
+    buildPhase = ''
+      runHook preBuild
+      export HOME="$(mktemp -d)"
+      cd frontend
+
+      mkdir yarnCache
+      yarn config set enableTelemetry 0
+      yarn config set cacheFolder yarnCache
+      yarn --cache-folder yarnCache
+      yarn config set --json supportedArchitectures.os '[ "linux", "darwin" ]'
+      yarn config set --json supportedArchitectures.cpu '["arm64", "x64"]'
+      runHook postBuild
+    '';
+    installPhase = ''
+      mkdir -p $out
+      cp -r yarnCache/v6 $out
+      cp -r node_modules $out
+    '';
+    dontConfigure = true;
+    dontCheckForBrokenSymlinks = true;
+    dontInstall = false;
+    dontFixup = true;
+    outputHashMode = "recursive";
+    doCheck = false;
+    outputHashAlgo = "sha256";
+    outputHash =
```

```suggestion
    dontConfigure = true;
    dontCheckForBrokenSymlinks = true;
    dontFixup = true;

    outputHashMode = "recursive";
    outputHash =
```
I think we can also omit dontConfigure and dontFixup but please test those and if they don't work please provide a short log why not. Also which symlink is broken? maybe we can just rm it?

---

yeah, I added the bypass the broken symlink check after it was added, firstly I thought the problem was on my end and wanted to keep investigating that, but then I realized that it was left by wepack-cli and forgot to remove it, my bad completely